City-of-Helsinki/berth-reservations-admin
# VEN-588 | Update boat card

## Description :sparkles:

- [x] Remove Boat, Large Boat and Application types duplication
- [x] Display `intendedUse` field in BoatCard for large boats
- [x] Display insurance and inspection certificates  in BoatCard for large boats
- [x] Remove boat fixture data duplication
- [x] Update boats card storybook
- [x] Move boats card tests under `__tests__`

### Closes :no_good_woman:
https://helsinkisolutionoffice.atlassian.net/browse/VEN-588

## Testing :alembic:
- Open customer with large boat: http://localhost:3000/customers/UHJvZmlsZU5vZGU6N2QyMGZlNmMtYWMzZC00ZTY0LWJkYzAtYTJiZjg5ODU5ZGU3
- See that intended use and certificate information is displayed as described in the ticket

### Automated tests :gear:️
Unit tests and React component snapshot test.
